### PR TITLE
Patch release `v0.4.6` :rocket:

### PR DESCRIPTION
**Pull request by eopb**

### CODE REVIEW

**Comment by conradludgate** [comment]
I got the tests passing in #101, it was quite awkward

File: reqwest-tracing/src/otel.rs [hunk 1]
```diff
@@ -144,6 +145,27 @@ mod test {
...
+
+            let telemetry = tracing_opentelemetry::layer().with_tracer(tracer);
+            let subscriber = Registry::default()
+                .with(
+                    filter::Targets::new().with_target("reqwest_tracing::otel::test", Level::DEBUG),
+                )
+                .with(telemetry);
+            tracing::subscriber::set_global_default(subscriber).unwrap();
+            global::set_text_map_propagator(TraceContextPropagator::new());
+        });
```
Comment (eopb) [inline reqwest-tracing/src/otel.rs:168]:
```suggestion
```

File: reqwest-tracing/src/otel.rs [hunk 2]
```diff
@@ -117,6 +117,7 @@ mod test {
     async fn make_echo_request_in_otel_context(client: ClientWithMiddleware) -> Response {
         static TELEMETRY: OnceLock<()> = OnceLock::new();
 
+        #[cfg(not(feature = "opentelemetry_0_20"))]
```
Comment (eopb) [inline reqwest-tracing/src/otel.rs:120]:
```suggestion
```

File: reqwest-tracing/Cargo.toml [hunk 3]
```diff
@@ -54,6 +54,7 @@ getrandom = { version = "0.2.0", features = ["js"] }
 tokio = { version = "1.0.0", features = ["macros"] }
 tracing_subscriber_0_2 = { package = "tracing-subscriber", version = "0.2.0" }
 tracing_subscriber_0_3 = { package = "tracing-subscriber", version = "0.3.0" }
+opentelemetry-stdout = { version = "0.1.0", features = ["trace"] }
```
Comment (eopb) [inline reqwest-tracing/Cargo.toml:57]:
```suggestion
```

File: reqwest-tracing/CHANGELOG.md [hunk 4]
```diff
@@ -4,7 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.4.6] - 2023-08-16
```
Comment (eopb) [inline reqwest-tracing/CHANGELOG.md:7]:
```suggestion
## [Unreleased]

## [0.4.6] - 2023-08-23
```

**Review by DarraghBlake — APPROVED** [review]

**Review by skos-ninja — APPROVED** [review]

**Pull request merged**